qt_binder/raw_widgets.py

```diff
@@ -693,7 +693,7 @@ def add_buttons_from_context(self, context):
 ]
 
 
-for obj in vars().values():
```

Comment:
What error is being avoided here? Mutation of the `dict` during iteration because of the assignment to `obj`?